external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchSourceSuite.scala

```diff
@@ -204,6 +204,43 @@ abstract class KafkaMicroBatchSourceSuiteBase extends KafkaSourceSuiteBase {
       StopStream)
   }
 
+  test("Rate limit set to Long.Max should not overflow integer " +
+    "during end offset calculation[SPARK-26718]") {
```

Comment:
Hi, @linehrr .
We usually put SPARK ID at the beginning of the test.
```scala
  test("SPARK-26718 Rate limit set to Long.Max should not overflow integer " +
    "during end offset calculation") {
```

Reply:
ok I got the idea that ticket number should be at the end from test case: `  test("ensure that initial offset are written with an extra byte in the beginning (SPARK-19517)") {`, but I can put it at the front.